From 7c1bae9cc4819ed9148354fd878824b40bd7a6a5 Mon Sep 17 00:00:00 2001
From: Stefan Saraev <stefan@saraev.ca>
Date: Sun, 6 Oct 2013 17:48:50 +0300
Subject: [PATCH] fix scrambled/hd channel scan

from wirbelscan:
  #define SCAN_TV        ( 1 << 0 )
  #define SCAN_RADIO     ( 1 << 1 )
  #define SCAN_FTA       ( 1 << 2 )
  #define SCAN_SCRAMBLED ( 1 << 3 )
  #define SCAN_HD        ( 1 << 4 )
---
 src/libxvdr/include/xvdr/dataset.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/libxvdr/include/xvdr/dataset.h b/src/libxvdr/include/xvdr/dataset.h
index 53b0540..36d9a94 100644
--- a/src/libxvdr/include/xvdr/dataset.h
+++ b/src/libxvdr/include/xvdr/dataset.h
@@ -287,8 +287,8 @@ public:
     FLAG_TV = 1,
     FLAG_RADIO = 2,
     FLAG_FTA = 4,
-    FLAG_SCRAMBLED = 16,
-    FLAG_HDTV = 32
+    FLAG_SCRAMBLED = 8,
+    FLAG_HDTV = 16
   } Flags;
 
   Verbosity verbosity;
-- 
1.7.2.5

